### PR TITLE
Fix some issues

### PR DESCRIPTION
**Pull request by yoichiro**
I would like to fix some issues:

* The "default_action" property in the "browser_action" is invalid.
* The "type" property for the jQuery.ajax calling would be wrong. I guess that the correct property is "dataType".

### CODE REVIEW

File: src/lib/qiita-client.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ const get = (path) => {
     jQuery.ajax({
       credentials: "include",
```
Comment (yoichiro) [inline src/lib/qiita-client.js]: Is this property "credentials" correct? I don't know this property...

Comment (r7kamura) [inline src/lib/qiita-client.js]: It looks wrong 😩 
This option was added for [fetch API](https://developer.mozilla.org/ja/docs/Web/API/GlobalFetch/fetch), but no longer needed now.

Comment (yoichiro) [inline src/lib/qiita-client.js]: I see. I intend to remove this line. Thanks.

**Comment by yoichiro** [comment]
@r7kamura It would be great if you can review this PR and merge it. Thank you in advance.

**Comment by r7kamura** [comment]
:+1:

**Pull request merged**